hosts.conf

```diff
@@ -0,0 +1,18 @@
+# Saps Deploy
+## Ansible user name on the hosts where the Saps component will be deployed (should be the same in all hosts)
```

Comment:
why do we need so many private keys in this declaration? Do we want to allow the user to use different keys? I guess the user would use the same key here, wouldn't?

Reply:
You're right. I modified it to use the same key for all hosts.